scripts/clang-format/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cd arangodb
+#find arangod -regex '.*\.\(cpp\|hpp\|cu\|c\|h\)' -exec clang-format -style=file -i {} \;
+find arangosh -regex '.*\.\(cpp\|hpp\|cu\|c\|h\)' -exec clang-format -style=file -i {} \;
+#find lib -regex '.*\.\(cpp\|hpp\|cu\|c\|h\)' -exec clang-format -style=file -i {} \;
+#find tests -regex '.*\.\(cpp\|hpp\|cu\|c\|h\)' -exec clang-format -style=file -i {} \;
```

Comment:
If this script is supposed to be run locally by developers, we should also add `enterprise` here, if it exists. So we should test for the existence of the `enterprise` subdirectory, and then apply formatting there too.